packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -408,7 +411,11 @@ export class SummarizerNodeWithGC extends SummarizerNode implements IRootSummari
 			return childNodesBaseGCDetails.get(id) ?? {};
 		};
 
-		const createDetails: ICreateChildDetails = this.getCreateDetailsForChild(id, createParam);
+		const createDetails: ICreateChildDetails = this.getCreateDetailsForChild(
```

Comment:
Unrelated: Too bad this logic seems to be duplicated between summarizerNode and summarizerNodeWithGc.